src/components/blogposts/BlogTeaser.js

```diff
@@ -1,9 +1,22 @@
+import {useState} from 'react';
 import styled from 'styled-components';
 
 export default function BlogTeaser() {
+	const [{data, error}, setPosts] = useState({data: '', error: null});
```

Comment:
your have to use data and error, if you dont use this variables your PR is not complete.